src/App.jsx

```diff
@@ -1,8 +1,14 @@
-import React from "react";
-import "./App.css";
+import "src/assets/styles/styles.scss";
+import Content from "./components/content/Content";
+import Navbar from "./components/navbar/Navbar";
```

Comment:
can we use relative import here ?
```suggestion
import Content from "components/content/Content";
import Navbar from "components/navbar/Navbar";
```